WISE-Community/WISE
# Issue 1933 box2dmodel converter

This PR includes code changes added to get certain functionality in the density project working.

**Test the project converter**
1. Download the density project from the WISE production library https://wise.berkeley.edu/teacher/library/tested
2. Add this line to your application.properties file and restart your local server. This is required because the converted project.json file is large and exceeds the default Tomcat 2MB post limit
```
server.tomcat.max-http-post-size=10MB
```
3. Import the WISE4 density project to your local WISE server
4. Open the WISE4 project in the Authoring Tool and convert it to WISE5
5. Make sure the project converts without error. The density models will not work because the [Box-2D-Model](https://github.com/WISE-Community/Box-2D-Model) folder needs to be added to the assets folder. There were also many other manual changes that were needed. We will test them in my converted project next.

**Test my converted density project**
1. Download the density project that I have converted and cleaned up
[144.zip](https://github.com/WISE-Community/WISE/files/3387728/144.zip)
2. Compare the WISE4 project with the WISE5 project and make sure
 - there are no missing steps
 - instructions/prompts are valid and accurate
 - density models work, save data, and reload data
 - table, graph, open response, multiple choice work is properly imported from one step to another
 - graphs work properly when they have lines and points on a single graph
 - branching works properly

Closes #1933
